.github/workflows/dynamo_bin_diff.yml

```diff
@@ -66,12 +87,9 @@ jobs:
       - name: Setup msbuild
         uses: microsoft/setup-msbuild@v2
       - name: Install dependencies for windows runtime
-        run: |
-          dotnet restore ${{ github.workspace }}\master\src\Dynamo.All.sln /p:Configuration=Release --runtime=win-x64 -p:DotNet=net8.0
+        run: dotnet restore ${{ github.workspace }}\master\src\Dynamo.All.sln /p:Configuration=Release --runtime=win-x64 -p:DotNet=net8.0
       - name: Build Dynamo with MSBuild
```

Comment:
Why is the `-p:DotNet=net8.0` not used in the previous `dotnet restore` in line 41?

Reply:
That is correct